src/coprocessor/codec/batch/lazy_column_vec.rs

```diff
@@ -20,29 +20,24 @@ use super::{LazyBatchColumn, VectorValue};
 use crate::coprocessor::codec::mysql::Tz;
 use crate::coprocessor::codec::Result;
 
-pub struct LazyBatchColumnVec<E> {
-    /// Multiple interested columns. Each column is either decoded, or not decoded.
+/// Stores multiple `LazyBatchColumn`s. Each column have equal length.
+#[derive(Clone, Debug)]
+pub struct LazyBatchColumnVec {
+    /// Multiple lazy batch columns. Each column is either decoded, or not decoded.
     ///
     /// For decoded columns, they may be in different types. If the column is in
     /// type `LazyBatchColumn::Encoded`, it means that it is not decoded.
     columns: Vec<LazyBatchColumn>,
-
-    /// The optional error occurred during batch execution. Once there is an error, this field
-    /// is set and no more rows should be fetched or executed.
-    some_error: Option<E>,
```

Comment:
So how do we handle error now?

Reply:
It is extracted to outside, will be introduced in later PRs. The core idea is to make LazyBatchColumnVec a pure Vec instead of something comprehensive so that LazyBatchColumnVec can be used in other places.